src/test/java/org/folio/mr/service/MediatedRequestActionsServiceTest.java

```diff
@@ -127,10 +140,12 @@ void sendItemInTransitSuccess() {
       .thenReturn(Optional.of(initialRequest));
     when(mediatedRequestsRepository.save(any(MediatedRequestEntity.class)))
       .thenReturn(updatedRequest);
-    when(inventoryService.fetchItem(initialRequest.getItemId().toString()))
-      .thenReturn(new Item());
     when(mediatedRequestMapper.mapEntityToDto(any(MediatedRequestEntity.class)))
       .thenReturn(mappedRequest);
+    when(searchClient.searchInstance(anyString()))
+      .thenReturn(new SearchInstancesResponse()
+        .instances(List.of(new SearchInstance().items(List.of(
+          new SearchItem().id(initialRequest.getItemId().toString()))))));
```

Comment:
Correct me if I'm wrong, but I believe the whole point of this fix is to be able to fetch an item from a different tenant. Do we actually do this in tests? If not, I really think we should.